packages/calcite-components/src/components/switch/switch.stories.ts

```diff
@@ -35,3 +35,14 @@ export const darkModeRTL_TestOnly = (): string => html`
 darkModeRTL_TestOnly.parameters = { modes: modesDarkDefault };
 
 export const disabled_TestOnly = (): string => html`<calcite-switch disabled checked></calcite-switch>`;
+
+export const Focus_TestOnly = (): string =>
+  html` <div>
+    <calcite-switch> </calcite-switch>
+    <script>
+      (async () => {
```

Comment:
Should this check for the switch being defined and not the date picker?

Reply:
side effects.